outport/process/executionOrder/transactionsExecutionOrder.go

```diff
@@ -173,7 +173,7 @@ func (s *sorter) getInvalidTxsExecutedInCurrentBlock(scheduledMbsFromPreviousBlo
 	for _, hash := range mb.TxHashes {
 		_, found := allScheduledTxs[string(hash)]
 		if found {
-			scheduledExecutedInvalidTxsHashesPrevBlock = append(scheduledExecutedInvalidTxsHashesPrevBlock, string(hash))
+			scheduledExecutedInvalidTxsHashesPrevBlock = append(scheduledExecutedInvalidTxsHashesPrevBlock, hex.EncodeToString(hash))
```

Comment:
is this still backward compatible?

Reply:
yes is not a problem. because this part is used only in indexer

Reply:
agree, did not noticed that the package was actually `outport...`